package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "setup-edm-action",
+  "version": "1.0.0",
+  "description": "GitHub Action for installing EDM",
+  "main": "dist/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
Q : Is this necessary or can it be removed?

Reply:
It is now removed and I added a `build` command instead.